source/mir/cv/core/simd.d

```diff
@@ -0,0 +1,77 @@
+/+
+    Utilities to help use SIMD vectors.
++/
+
+module mir.cv.core.simd;
```

Comment:
1. Yes
2. Yes, but to partially, and to mir-algorithm, because it is compile time logic. We need lists of available vector length for different types.